examples/layers/preprocessing/segmentation/demo_utils.py

```diff
@@ -44,7 +44,6 @@ def to_dict(datapoint):
     input_mask = tf.one_hot(
         tf.squeeze(tf.cast(input_mask, tf.int32), axis=-1), depth=3
     )
-    input_image = tf.transpose(input_image, (2, 0, 1))
```

Comment:
What is the purpose of dropping this? Does it break other segmentation demos?

Reply:
When transposing an image, cutmix operation does not work. Also, I checked with other segmentation demos, it works perfectly too.